Kiosk/App/AppDelegate.swift

```diff
@@ -16,6 +16,10 @@ public class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication!, didFinishLaunchingWithOptions launchOptions: NSDictionary!) -> Bool {
 
+        // Disable sleep timer
+        UIApplication.sharedApplication().idleTimerDisabled = true
```

Comment:
I know, I know, but I'm just a selfish actor in a dysfunctional (operating) system.